.github/workflows/ci.yml

```diff
@@ -55,6 +55,8 @@ jobs:
         poetry run ./typecheck.sh
 
     - name: Python tests
+      env:
+        WK_TOKEN: ${{ secrets.WK_TOKEN }}
```

Comment:
what kind of token is used here? does it belong to a separate dummy organization on webknossos.org? Alternatively, the tests could be hooked up with https://master.webknossos.xyz/

Reply:
Yes, this is for a separate dummy user on webknossos.org, with it's own Test orga. I've no strong opinion if we rather should use https://master.webknossos.xyz, as long as they are in sync. If we hold back a version from deployment using wk.org might be the better option. Also, wk.org has extra datastores, which is helpful for demos and testing. /cc @fm3 Do you have an opinion on this?

Reply:
I think both instances should only be used for things that have no side effects, though errors in the tests (or webknossos’ handling of them) may be more damaging on webknossos.org. I’d tend to prefer the non-production instance for tests, in order to reduce load and risk, but I did not read the code here so it may be safe enough.

Note that master.webknossos.xyz is updated nightly to the current master branch, while wk.org is *usually* updated directly after merge, but is sometimes rolled back if there is a problem with the master, or sometimes the deployment is held back a few days if we want to test on other instances first.

Reply:
Since the test suite is likely to grow in the future, a strict separation from production instances sounds like a good strategy to me (with respect to load, risk, resources/garbage etc.).

> as long as they are in sync.

We could probably setup a dev instance which always has the same version as wkorg.

> Also, wk.org has extra datastores, which is helpful for demos and testing. 

Ideally, we would extend the dev instance so that it also has these things. However, I can understand that some effort is needed for this.

I'd be fine with leaving it as is for now, but in the long-term picture I'd prefer the separation explained above.

Reply:
Sound's good, I'll adapt this in one of the next PRs